content/_partials/carousel.html.haml

```diff
@@ -21,6 +21,14 @@
   })
 
 %div#ji-home-carousel.carousel.slide{'data-ride' => 'carousel',:class => page.topic}
+#ji-download
```

Comment:
I am sorry. My bad! :(

On Wed, Mar 23, 2016 at 9:50 PM, R. Tyler Croy <notifications@github.com>
wrote:

> In content/_partials/carousel.html.haml
> <https://github.com/jenkins-infra/jenkins.io/pull/156#discussion_r57189616>
> :
>
> > @@ -21,6 +21,14 @@
> >    })
> >
> >  %div#ji-home-carousel.carousel.slide{'data-ride' => 'carousel',:class => page.topic}
> > +#ji-download
>
> @arshdkhn1 <https://github.com/arshdkhn1> I didn't catch this in the
> original review, the site compilation failure you posted to the
> jenkinsci-dev@ mailing list is due to the indentation of this Haml.
>
> Haml uses indentation to know how to nest tags when it renders, so the
> .btn-group here is too far indented under #ji-download.
>
> I'm fixing this in a follow-up pull request right now
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/156/files/a3a9b7cb46e25d577ce708d9cfc5e1f738ed283f#r57189616>
>